app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -368,31 +422,76 @@ class Identification extends React.Component {
               <Form.Row className="pt-1">
                 <Form.Group as={Col} md="auto">
                   <Form.Label className="nav-input-label">RACE (SELECT ALL THAT APPLY)</Form.Label>
-                  <Form.Check type="switch" id="white" label="WHITE" checked={this.state.current.patient.white} onChange={this.handleChange} />
+                  <Form.Check
+                    type="checkbox"
+                    id="white"
+                    label="WHITE"
+                    checked={!!this.state.current.patient.white}
+                    disabled={!!this.state.current.patient.race_unknown || !!this.state.current.patient.race_refused_to_answer}
```

Comment:
I'd suggest extracting:
```
{!!this.state.current.patient.race_unknown || !!this.state.current.patient.race_refused_to_answer}
```
into a variable at the start rather than having the same logic across all other race checkboxes.

OR actually even better - instead of having separate checkbox elements explicitly listed out here, can we just have a loop that loops over all of the possible races and dynamically creates the `Form.Check`? I think that should be pretty straightforward to do and reduce many many lines of code and the possibility for typos/mistakes.

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/pull/689/commits/767b8765f1830cb0e34e8bc8b1adbbe48c0786f5#diff-0b38c12b7f12dd9f316a576d5b5dc6c12f0b8019720598550b239fca8828d292R411